### ISSUE TITLE
App crashes in Production

### ISSUE THREAD

**Issue by lutakyn** [Tealium/tealium-react-native#60]
Tealium has been crashing in production in Play Store. Crashes are affecting mostly Android 9 and Android 10 OS users.

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performPauseActivityIfNeeded (ActivityThread.java:4863)
  at android.app.ActivityThread.performPauseActivity (ActivityThread.java:4814)
  at android.app.ActivityThread.handlePauseActivity (ActivityThread.java:4766)
  at android.app.servertransaction.PauseActivityItem.execute (PauseActivityItem.java:46)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:176)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:97)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2216)
  at android.os.Handler.dispatchMessage (Handler.java:107)
  at android.os.Looper.loop (Looper.java:237)
  at android.app.ActivityThread.main (ActivityThread.java:7948)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1075)
Caused by: java.lang.NullPointerException: 
  at com.tealiumreactnative.TealiumModule$1.onHostPause (TealiumModule.java:418)
  at com.facebook.react.bridge.ReactContext.onHostPause (ReactContext.java:240)
  at com.facebook.react.ReactInstanceManager.moveToBeforeResumeLifecycleState (ReactInstanceManager.java:673)
  at com.facebook.react.ReactInstanceManager.onHostPause (ReactInstanceManager.java:497)
  at com.facebook.react.ReactInstanceManager.onHostPause (ReactInstanceManager.java:519)
  at com.facebook.react.ReactDelegate.onHostPause (ReactDelegate.java:62)
  at com.facebook.react.ReactActivityDelegate.onPause (ReactActivityDelegate.java:93)
  at com.facebook.react.ReactActivity.onPause (ReactActivity.java:50)
  at android.app.Activity.performPause (Activity.java:8142)
  at android.app.Instrumentation.callActivityOnPause (Instrumentation.java:1508)
  at android.app.ActivityThread.performPauseActivityIfNeeded (ActivityThread.java:4853)
```
![tealiumerror4](https://user-images.githubusercontent.com/32960420/94403953-bb3f3480-015d-11eb-9b77-ee37dacb5cc0.png)

Environment
"react": "16.9.0",
 "react-native": "0.61.1",
"tealium-react-native": "^1.0.9",
Android Target SDK : 28

**Comment by jameskeith** [comment]
Hi @lutakyn 
Thanks for bringing this to our attention - looking through the 1.0.9 source, this would be thrown because of a failure to return a non-null Lifecycle instance:
https://github.com/Tealium/tealium-react-native/blob/1.0.9/npm-package/android/src/main/java/com/tealiumreactnative/TealiumModule.java#L418

It looks like this has already been addressed in v1.1.0 though, so can I suggest updating to v1.1.0 to resolve your issue please?
https://github.com/Tealium/tealium-react-native/blob/1.1.0/npm-package/android/src/main/java/com/tealiumreactnative/TealiumModule.java#L456

**Comment by lutakyn** [comment]
Library upgraded and I believe the fix will be applied.I haven't tested it yet.Difficult to reproduce the same error.Closing the ticket
